fusedrug/data/tokenizer/ops/modular_tokenizer_ops.py

```diff
@@ -148,8 +148,8 @@ def get_min_max_sentinels(
 
         return (min_token, max_token)
 
-    def get_token_id(self, token_str: str) -> int:
-        ans = self._tokenizer.token_to_id(token_str)
+    def get_token_id(self, token_str: str, t_type: Optional[str] = None) -> int:
```

Comment:
What is this t_type?
Is it something we need only to follow huggingface interface?

Reply:
@SagiPolaczek - add the following documentation here (taken from modular tokenizer):
t_type (:obj:`str`): The subtokenizer to use. If None, the first (in order defined in the config)
                subtokenizer is used. If the token is special, type should not be set. TODO: raise a warning
                if type=None and the token is not special
@mosheraboh - it's the tokenizer type, not something that exists in huggingface, but something useful for get_token_id, since we may have tokens that may get different IDs (e.g. 'C' can be carbon in SMILES or cysteine in AAs). This is usually not a problem, because we mainly use this for special tokens (and they have consistent IDs across all subtokenizers), but in case we use it with regular tokens, this'll help it behave in a more predictable way

Reply:
Cool, thanks for the explnation